recipes/opentracing-cpp/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class OpenTracingConan(ConanFile):
+    name = "opentracing-cpp"
+    description = "C++ implementation of the OpenTracing API http://opentracing.io"
+    license = "Apache 2.0 License"
+    topics = ("conan", "opentracing")
+    homepage = "https://github.com/opentracing/opentracing-cpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_mocktracer": [True, False],
```

Comment:
Please rename to `enable_mocktracer` (or something similar).

Reply:
Done, thanks for reviewing!